test/e2e/functional/tests/e2e.go

```diff
@@ -183,9 +184,9 @@ var _ = Describe("metallb", func() {
 
 			assertConfigMapMatchYAML("{}")
 		},
-			table.Entry("Test AddressPool object with default auto assign", "addresspool1", &metallbv1beta1.AddressPool{
+			table.Entry("Test AddressPool object with default auto assign", "test-addresspool1", &metallbv1beta1.AddressPool{
```

Comment:
I'm wondering why you added a prefix to the addresspools names since the cleanup is anyway done by the namespace

Reply:
It's for keeping the option to run e2e parallel to MetalLB deployed, this way we could recognize test resources and delete them specifically.